src/cli.ts

```diff
@@ -96,6 +96,12 @@ yargs
           type: 'boolean',
           describe: 'exit with non-zero code if there are existing upgrades',
         })
+        .option('showAll', {
```

Comment:
```suggestion
        .option('all', {
```
this would do I think